doc/ref/matrix.xml

```diff
@@ -614,6 +615,15 @@ grease and level <A>b</A> blocking.
 <#Include Label="BlockMatrix">
 <#Include Label="MatrixByBlockMatrix">
 
+</Section>
+
+
+<!-- %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% -->
+<Section Label="Linear Programming">
+<Heading>Linear Programming</Heading>
+
+<#Include Label="SimplexMethod">
```

Comment:
This PR changes two functions from undocumented to documented; that definitely warrants a release notes entry